### ISSUE TITLE
CreationProperties property doesn't work when set in XAML without StaticResource

### ISSUE THREAD

**Issue by champnic** [MicrosoftEdge/WebView2Feedback#552]
I tried CreationProperties, but that doesn't seem to work with setting Source in XAML as well:
`<wv2:WebView2 Name="webView" Height="500"  Source="https://www.whatismybrowser.com/">
    <wv2:WebView2.CreationProperties>
        <wv2:CoreWebView2CreationProperties UserDataFolder="C:\temp\edgechromium"/>
    </wv2:WebView2.CreationProperties>
</wv2:WebView2>`

_Originally posted by @priyasgs in https://github.com/MicrosoftEdge/WebView2Feedback/issues/195#issuecomment-712865186_

[AB#29901175](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/29901175)

**Comment by champnic** [comment]
I've confirmed in a regular WPF app that we have a bug with CreationProperties, where if it gets defined in the XAML as you have it it doesn't work. It does work if instead you define the CreationProperties as a resource:
```
    <Window.Resources>
        <wv2:CoreWebView2CreationProperties UserDataFolder="C:\temp\myFolder" 
                                            x:Key="MyCreationProperties" />
    </Window.Resources>
    <wv2:WebView2
            CreationProperties="{StaticResource MyCreationProperties}"
            .... other attributes />
```

My guess is it's related to the order of object creation from the XAML - the Source property gets applied and begins implicit initialization before the CoreWebView2CreationProperties have been created and set to the CreationProperties property. I verified this by setting the Source after the CreationProperties, which works fine:
```
    <wv2:WebView2
            Name="webView"
            Visibility="Visible"
            CoreWebView2Ready="webView_CoreWebView2Ready"
            NavigationStarting="webView_NavigationStarting"
            NavigationCompleted="webView_NavigationCompleted"
            Loaded="webView_Loaded">
        <wv2:WebView2.CreationProperties>
            <wv2:CoreWebView2CreationProperties UserDataFolder="C:\temp\myFolder"/>
        </wv2:WebView2.CreationProperties>
        <wv2:WebView2.Source>https://bing.com</wv2:WebView2.Source>
    </wv2:WebView2>
```

**Comment by ukandrewc** [comment]
Got a similar issue with Winforms, trying to test fixed distribution.
Getting an error in CoreWebView2InitializationCompleted

Initialization code is:
```
Dim Props As New Microsoft.Web.WebView2.WinForms.CoreWebView2CreationProperties
Props.BrowserExecutableFolder = $"{My.Application.Info.DirectoryPath}\WebView2_x86\"
NewBrowser = New WebView2
NewBrowser.Dock = DockStyle.Fill
NewBrowser.CreationProperties = Props
```
**e.InitializationException.InnerException.Message shows:**

![image](https://user-images.githubusercontent.com/30246320/100844091-40eba300-3473-11eb-83f6-55bbbf9513c5.png)

It's the same if I try and initialize using EnsureCoreWebView2Async

Thanks

**Comment by champnic** [comment]
@ukandrewc What version of the runtime and SDK are you using? This looks like a potential mismatch between them (or you are running into the same root cause as #678)

**Comment by ukandrewc** [comment]
@champnic Sorry forgot that bit, Version 89.0.723.0 (Official build) dev (32-bit)  V1.0.707 pre-release.

There's no rush for me, I was only starting to test fixed distribution. Would it be fixed in 1.0.721?

**Comment by champnic** [comment]
@ukandrewc This should be fixed in 1.0.781-prerelease. Thanks!

**Comment by ukandrewc** [comment]
@champnic  & Co: Cheers!
